src/platform/packages/shared/kbn-react-field/src/field_icon/field_icon.tsx

```diff
@@ -46,6 +46,8 @@ export const typeToEuiIconMap = {
   rank_feature: { iconType: 'tokenRankFeature' },
   rank_features: { iconType: 'tokenRankFeatures' },
   histogram: { iconType: 'tokenHistogram' },
+  exponential_histogram: { iconType: 'tokenHistogram' },
+  tdigest: { iconType: 'tokenHistogram' },
```

Comment:
???

I don't see any other options

Reply:
I think it is ok to use the same. These 2 field types are connected with histograms.  ES is going to add support for the existing histogram type in ESQL by casting to one of these